magic_the_gathering/turn.py

```diff
@@ -0,0 +1,42 @@
+from magic_the_gathering.game_state import GameState
+from magic_the_gathering.phases.combat_beginning import CombatBeginningPhase
+from magic_the_gathering.phases.combat_damage import CombatDamagePhase
+from magic_the_gathering.phases.combat_declare_attackers import CombatDeclareAttackersPhase
+from magic_the_gathering.phases.combat_declare_blockers import CombatDeclareBlockersPhase
+from magic_the_gathering.phases.combat_end import CombatEndPhase
+from magic_the_gathering.phases.draw import DrawPhase
+from magic_the_gathering.phases.end import EndPhase
+from magic_the_gathering.phases.main import MainPhase
+from magic_the_gathering.phases.untap import UntapPhase
+from magic_the_gathering.phases.upkeep import UpkeepPhase
+
+
+class Turn:
+    def __init__(self):
+        self.__phases = [
+            UntapPhase(),
+            UpkeepPhase(),
+            DrawPhase(),
+            MainPhase(name="Main Phase 1"),
```

Comment:
Btw your precombat main phase and postcombat main phase names were good too, as you prefer

Reply:
What is the most standard way to call these phases in Magic?